src/cms/models/pages/page_translation.py

```diff
@@ -119,6 +119,20 @@ def combined_text(self):
             return attached_text + self.text
         return self.text + attached_text
 
+    @property
+    def combined_last_updated(self):
+        """
+        This function combines the last_updated date from this PageTranslation and from a mirrored page.
+        If this translation has no content, then the date from the mirrored translation will be used. In
+        other cases, the date from this translation will be used.
+
+        :return: The last_upated date of this or the mirrored page translation
+        :rtype: ~django.db.models.DateTimeField
+        """
+        if self.page.get_mirrored_page is not None and not self.text:
+            return self.page.get_mirrored_page(self.language.code).last_updated
```

Comment:
I noticed that `get_mirrored_page()` uses `get_translation()` internally:
https://github.com/Integreat/integreat-cms/blob/000f2abdebd49ef17d13d7640c3b96f2413ca44c/src/cms/models/pages/page.py#L140
While you're at it, would it make sense to change that to `get_public_translation()`?

Reply:
Yes, I can do that.